pkgs/sketch_pad/pubspec.yaml

```diff
@@ -11,28 +11,39 @@ dependencies:
   dartpad_shared: any
   flutter:
     sdk: flutter
+  flutter_localizations:
+    sdk: flutter
   fluttering_phrases: ^1.0.0
   go_router: ^12.1.0
   http: any
+  intl: any
   json_annotation: ^4.8.1
   pointer_interceptor: ^0.9.0
   provider: ^6.0.0
   split_view: ^3.2.0
   url_launcher: any
   url_strategy: any
   vtable: ^0.4.0
+  messages: ^0.1.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
   flutter_lints: ^3.0.0
   test: any
+  build_runner: 2.4.6
+  build_runner_core: 7.2.10
+  flutter_messages: ^0.1.0
+  messages_serializer: ^0.1.0
+  messages_builder: ^0.1.0
 
 flutter:
   uses-material-design: true
   assets:
     - assets/dart_logo_128.png
     - assets/flutter_logo_192.png
+    - lib/l10n/en.json
```

Comment:
Not for this PR, but we should add some docs to the repo about how to contribute a new localization; this will be a good way to get more people involved.